tests/cgroupstats_tests.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from gnlpy.cgroupstats import CgroupstatsClient
+
+import re
+import traceback
+import unittest
+
+
+CGROUP_PATH = '/proc/self/cgroup'
+CGROUP_CPU_PATH = '/sys/fs/cgroup/cpu/'
+
+
+class CgroupstatsTestCase(unittest.TestCase):
+
+    def setUp(self):
+        # check for presence of cpu cgroup so we know what to assert for
+        try:
+            with open(CGROUP_PATH, mode='r') as f:
+                file_contents = f.read()
+                if re.search('[,:]cpu[,:]', file_contents) is None:
+                    self.cgroup_available = False
+                else:
+                    self.cgroup_available = True
+        except IOError:
+            self.cgroup_available = False
+
+    def test_get_cgroup_stats(self):
```

Comment:
I am tempted to think that the systems where those tests are going to run will support cgroups.
I don't think we should prevent the test from running if cgroups are not available.
Currently, I believe they are on travis, which is where the test are going to run and signal any issue.
Individual contributors can just not run this test suite if it is not supported on their setup.

Worst case, we could use `expectedFailure` decorator
https://docs.python.org/2/library/unittest.html#unittest.expectedFailure

But I rather have a test fail because of environment change than ignoring the test altogether without properly signaling it.